src/server.ts

```diff
@@ -10,10 +10,12 @@ import { resolvers } from "./resolvers";
 import { typeDefs } from "./schema";
 import { PostgresConnectionOptions } from "typeorm/driver/postgres/PostgresConnectionOptions";
 import { PostgresConnectionCredentialsOptions } from "typeorm/driver/postgres/PostgresConnectionCredentialsOptions";
+import { formatError } from "./types-and-classes/errors";
 
 export const server = new ApolloServer({
   typeDefs,
   resolvers,
+  formatError: formatError,
```

Comment:
Aqui dá pra simplificar (quando a variável tem o msm nome do campo)

```suggestion
  formatError,
```